src/main/java/uk/gov/justice/digital/hmpps/cmd/api/config/QuartzConfiguration.kt

```diff
@@ -0,0 +1,233 @@
+package uk.gov.justice.digital.hmpps.cmd.api.config
+
+import org.quartz.JobDetail
+import org.quartz.Trigger
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration
+import org.springframework.context.ApplicationContext
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
+import org.springframework.core.io.ClassPathResource
+import org.springframework.scheduling.quartz.CronTriggerFactoryBean
+import org.springframework.scheduling.quartz.JobDetailFactoryBean
+import org.springframework.scheduling.quartz.SchedulerFactoryBean
+import org.springframework.scheduling.quartz.SpringBeanJobFactory
+import uk.gov.justice.digital.hmpps.cmd.api.controllers.*
+import javax.sql.DataSource
+
+@Configuration
+@Profile("!dev")
```

Comment:
is this standard practise?

Reply:
I think its fine because we want it to run on the other profiles by default, if anything a 'dev' configuration is missing (which would be a copy of this file with the ! taken out), the reason we don't want it to run in dev is the postgres driver in the quartz.properrties which needs to be H2.